dhis-2/dhis-api/src/main/java/org/hisp/dhis/sms/config/SmsGatewayConfig.java

```diff
@@ -103,13 +103,18 @@ public void setUid( String uid )
         this.uid = uid;
     }
 
-    @JsonIgnore
-    public String getPassword()
+    public String getValuePassword()
     {
         return password;
     }
 
-    public void setPassword(String password)
+    @JsonProperty
```

Comment:
We discussed this on Slack. So I will leave it to the decision you make after discussion with Lars.